rustls/src/msgs/persist.rs

```diff
@@ -229,7 +241,7 @@ impl ClientSessionCommon {
 
     /// Test only: wind back epoch by delta seconds.
     #[doc(hidden)]
-    pub fn rewind_epoch(&mut self, delta: u32) {
+    fn rewind_epoch(&mut self, delta: u32) {
```

Comment:
I would be inclined to just inline this. If not, can lose the `#[doc(hidden)]` since it's now private anyway.

Reply:
>  If not, can lose the #[doc(hidden)] (and comment) since it's now private anyway.

Nice catch +1